### DISCUSSION TITLE
Starting the conversation about Addons

### DISCUSSION THREAD

**Discussion by SquatchHunter** [BedrockTweaks/Files#134] [Ideas]
Vanilla Tweaks has updated to 1.21 and with it a few packs have been removed, a few packs added and some made obsolete (or I am guessing they are) and they kind of redid the entire layout for categories so some have moved.

I will start with packs they added:
- Cauldron Mud (Cauldron Concrete was moved, but existed)
- Ender Chest Always Drops
- Glass Always Drops
- More Effective Tools
- Storm Channeling
- Husks Drop Sand
- Silk Touch Budding Amethyst
- Wandering Trader Announcements

Packs removed:
- Pillager Tools (probably in favour of Ominous Bottles)
- Larger Phantoms
- Count Mob Deaths
- Item Averages
- Gem Villagers
- Treasure Gems
- Tag
- Thunder Shrine (Storm Channeling replaced?)

Packs renamed:
- XP Management > XP Bottling

Now onto categories, the new structure is;
- Decorative/Cosmetic
- Convenience
- Gameplay Changes
- Informative
- Teleport Commands
- Admin Tools

I propose we add one for 'Fun' packs, we _should_ be able to bring back packs like Tag and maybe add some of our own :)

**Comment by gyver5681** [comment]
New guy here, I don't want to go too much off topic here, so I'll be as brief as I can. Once I get momentum on a project, I have to keep going at it full tilt or I loose interest and move on to something else (ADHD). That said, What's going on with this? Evidently the talk has been going on for a while, since this idea was posted in July with no responses. What challenges are being faced regarding getting an addon's section going? Is there some reason we can't just release individual packs and let people download them? What's the deal? I want to help, but I can't do anything unless actionable items are presented.


We also don't need the one-player-sleep addon any more, that's vanilla functionality now.

**Reply by PAPISOP** [reply]
Addon is in active development by @drav0011 to implement them correctly in the website.

The challenges we faced were the updating the resource packs. Our main goal was to make the resource packs to be done and be as same as Vanilla Tweaks. As you know, the resource packs were a mess and it took us 4-5 months to properly implement resource packs by fixing and updating every resource packs and the major addition was addition of combinations. We are still not done yet as there are resource packs like Transparent UI which is possible in bedrock but it is not done yet. This resource pack will be done after addons release.

The problem with uploading individual addon is that we have to follow a design which is going to be made by @drav0011. Currently DrAv hasn't finalized the design yet and without a design, we can't make the resource packs for the website which requires specific design, and a design also ensures that newer contributors can also modify the addon codes easily if the original creator is absent.

About multiplayer sleep addon, it's still up for debates and I think we might or might not add it, but it does add additional features.

We are waiting for @drav0011 call when he adds the first addon with the finalized design to the repo by #328
